website/package.json

```diff
@@ -62,7 +60,6 @@
     "next-auth": "^4.20.1",
     "next-i18next": "^13.2.2",
     "nodemailer": "^6.9.1",
-    "npm": "^9.6.2",
```

Comment:
`npm` is meant to be installed globally, so there is no need to include it in `package.json`